package.json

```diff
@@ -64,5 +64,8 @@
   "lint-staged": {
     "*.{js,ts,tsx}": "eslint --cache --fix --ext .js,.ts,.tsx --max-warnings 0",
     "*.{js,json,ts,tsx,css,md,mdx}": "prettier --write"
+  },
+  "resolutions": {
```

Comment:
super annoying that I had to do this, but a deep dependency inside react-scripts brings an older version of rollup and ruins types without this line. thinking of a better fix.